src/wasmtime/crates/wasmtime/src/runtime/store.rs

```diff
@@ -1326,6 +1336,38 @@ impl<'a, T> StoreContextMut<'a, T> {
         self.0.rewinding = state;
     }
 
+    // store the unwind data, return its hash
+    pub fn store_unwind_data(&mut self, ptr: *const u8, len: usize) -> u64 {
+        // Allocate a vector with enough capacity to hold the data.
```

Comment:
Do we need to check the ptr is not null here?

Reply:
This is invoked by wasmtime and ptr is something created by wasmtime instead of a user-passed argument. ptr is not supposed to be zero since it is basically the base_address of the liear memory adding some potential offsets.

Reply:
Leaving that explanation in a comment would be nice